fda-model/compute/simulation.cs

```diff
@@ -480,6 +480,23 @@ public SimulationBuilder withLevee(UncertainPairedData upd, double topOfLeveeEle
                 _sim._topOfLeveeElevation = topOfLeveeElevation;
                 return new SimulationBuilder(_sim);
             }
+            public SimulationBuilder withLevee(bool useDefaultFunction, double topOfLeveeElevation)
```

Comment:
im not fond of this - this is essentially something that shouldnt live in the model itself. it is something that deals with making it easier for a user (not a programmer) to interact with the model so it should be confined to the user interface or view model.

Reply:
understood. and we should, in the right place.